Update custom elements web platform tests

rniwa opened the pull request:
#### 6be69c8a257d2347a93caa69a9d9a9b875579ecf
<pre>
Update custom elements web platform tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=302435">https://bugs.webkit.org/show_bug.cgi?id=302435</a>

Reviewed by Anne van Kesteren.

Update custom elements web platform test as of d29d199654b6bcb04c39901066e99702dec35135.

* LayoutTests/imported/w3c/web-platform-tests/custom-elements/behaves-like-button-with-popover.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/behaves-like-button-with-popover.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/behaves-like-button.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/behaves-like-button.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/Element-innerHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/Element-innerHTML.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/ShadowRoot-init-customElementRegistry-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/ShadowRoot-init-customElementRegistry.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/ShadowRoot-init-declarative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/ShadowRoot-init-declarative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/element-mutation-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/element-mutation.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/scoped-registry-define-upgrade-criteria-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/scoped-registry-define-upgrade-criteria.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/valid-custom-element-names-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/valid-custom-element-names.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/303038@main">https://commits.webkit.org/303038@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1c1103039150321a5bc6ac155df218be5da7e763

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/130784 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/159/builds/3107 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/41743 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/138210 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/82434 "Built successfully") | ⏳ 🛠 ios-apple 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/3087 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/156/builds/2950 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/99644 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/67487 "Passed tests") | ⏳ 🛠 mac-apple 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/133730 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/162/builds/2217 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/117105 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/80345 "Passed tests") | | ⏳ 🛠 vision-apple 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/154/builds/2141 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/35234 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/81462 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/110732 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/35738 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/140686 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/153/builds/2851 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/161/builds/2571 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/108151 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/160/builds/2897 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/113438 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/108079 "Passed tests") | | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/164/builds/2182 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/31864 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/55853 "Built successfully") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/20393 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/2919 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/157/builds/2740 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/163/builds/2940 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/152/builds/2847 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

annevk reviewed (approved):
Looks good modulo bots.

rniwa committed:
Update custom elements web platform tests
https://bugs.webkit.org/show_bug.cgi?id=302435

Reviewed by Anne van Kesteren.

Update custom elements web platform test as of d29d199654b6bcb04c39901066e99702dec35135.

* LayoutTests/imported/w3c/web-platform-tests/custom-elements/behaves-like-button-with-popover.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/behaves-like-button-with-popover.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/behaves-like-button.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/behaves-like-button.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/Element-innerHTML-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/Element-innerHTML.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/ShadowRoot-init-customElementRegistry-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/ShadowRoot-init-customElementRegistry.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/ShadowRoot-init-declarative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/ShadowRoot-init-declarative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/element-mutation-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/element-mutation.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/scoped-registry-define-upgrade-criteria-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/scoped-registry-define-upgrade-criteria.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/valid-custom-element-names-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/valid-custom-element-names.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/registries/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/w3c-import.log:

Canonical link: https://commits.webkit.org/303038@main
